### PR TITLE
clarify that "Leave" refers to guild/party; fix pluralisation in keep/remove challenge tasks

### PR DESCRIPTION
**Pull request by Alys**
This changes the "Leave" button on groups to "Leave Guild" or "Leave Party" as discussed with @Tressley and @lemoness last week. This clarifies that the button is not referring to the challenges that are just above it.

image
image

This also changes the "Keep It" / "Remove It" button when leaving a challenge to "Keep/Remove Them" because the buttons are referring to all the challenge tasks.
current production site:
image
New code:
image

### CODE REVIEW

File: website/client/components/groups/group.vue [hunk 1]
```diff
@@ -105,7 +105,7 @@
       .section(v-if="sections.challenges")
         group-challenges(:groupId='searchId')
     div.text-center
-      button.btn.btn-danger(v-if='isMember', @click='clickLeave()') {{ $t('leave') }}
```
Comment (paglias) [inline website/client/components/groups/group.vue:108]: I think this string is only used there so we can remove it

**Review by paglias — CHANGES_REQUESTED** [review]

**Comment by Tressley** [comment]
Small addition, the button strings for leaving a Challenge should be "Keep Tasks" and "Remove Tasks."

File: website/client/components/groups/group.vue (hunk 1 above)
Comment (Alys) [reply website/client/components/groups/group.vue:108]: It's still used in buttons on the guild list pages:
img

**Review by Alys — COMMENTED** [review]
I've changed  "Keep Them" and "Remove Them" to "Keep Tasks" and "Remove Tasks".

**Review by paglias — APPROVED** [review]

**Pull request merged**